--- a/feeds/luci/modules/luci-base/ucode/sys.uc	2024-10-28 10:23:18.997941894 +0800
+++ b/feeds/luci/modules/luci-base/ucode/sys.uc	2024-10-28 11:04:48.251907738 +0800
@@ -8,7 +8,7 @@
 	let line, list = [];

 	for (let line = top.read('line'); length(line); line = top.read('line')) {
-		let m = match(trim(line), /^([0-9]+) +([0-9]+) +(.+) +([RSDZTWI][<NW ][<N ]) +([0-9]+m?) +([0-9]+%) +([0-9]+%) +(.+)$/);
+		let m = match(trim(line), /^([0-9]+) +([0-9]+) +(.+) +([RSDZTWI][<NW ][<N ]) +([0-9]+m?) +([0-9]+%) +([0-9]+) +([0-9]+%) +(.+)$/);

 		if (m && m[8] != '/bin/busybox top -bn1') {
 			push(list, {
@@ -18,8 +18,8 @@
 				STAT: m[4],
 				VSZ: m[5],
 				'%MEM': m[6],
-				'%CPU': m[7],
-				COMMAND: m[8]
+				'%CPU': m[8],
+				COMMAND: m[9]
 			});
 		}
 	}
